CHANGELOG.md

```diff
@@ -1,5 +1,87 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.4.0
+
+<!-- generated comparing v5.3.1..master -->
+
+_Jan 31, 2022_
+
+A big thanks to the 22 contributors who made this release possible. Here are some highlights ✨:
+
```

Comment:
```suggestion
@siriwatknp wrote an [article about the introduction of callback support in style overrides](/blog/callback-support-in-style-overrides/) (#30668)
```

Reply:
Hmmm, I think we should just tweet about it. Feels wrong to promote it in the next release updates :)

Reply:
It would be a plus one channel to promote blog content, to complement Twitter a bit. I think it's worth it :)

Reply:
IMO the changelog should be as succinct as possible. Developers rely on it to decide whether they can safely update the version of MUI in their projects, and not necessarily to learn about changes to the documentation or our website. I wouldn't treat it as a marketing tool.
TBH I would even remove the highlight parts and leave just the summary of changes.

Reply:
I'll keep the highlights to make it consistent with the previous changelogs. I would say highlights are good, if you don't have time to go trough all changes to see if you are interested to update. In this case is about a documentation, so it's not really useful, but I see your point. Will leave it as is for now.

Reply:
:+1: To clarify: I didn't mean removing the highlights from this specific release, but to consider not having them at all. Twitter/blog and even GitHub releases could be a better place for them.

Reply:
Ah ok, yeah worth discussing it with the team :)